themes/digital.gov/layouts/partials/core/featured-resource.html

```diff
@@ -1,66 +1,104 @@
-{{/* Featured Resource Component
+<!-- prettier-ignore-start -->
+{{/* 
 
-  @params {string} resourcePath - A relative hugo path to lookup a page to feature, use a forward slash before the path and no trailing slash.
-  @params {string} kicker - Text that is displayed above the resources main content.
+  Featured Resource Component displays an internal or external resource wrapped in a gray rounded box.
+  If the link is internal the hugo page lookup will return the fields to display.
+  If the link is external then yaml data or shortcode fields will be used to populate the display fields.
+  The source data can be from a hugo page, yaml data from a topics page frontmatter or a shortcode used in markdown.
 
-  @example shortcode
-  {{< featured-resource resourcePath = "/guides/hcd/discovery-concepts"
->}}
+  @params {object} resource_data - a dictionary object to display a resource
+    - link (required) - can be internal hugo path or external http(s) link
+    - title - the title of the resource
+    - summary - the short description for the resource
+    - kicker - a short label or message at the top of the resource box
 
-
-<!-- prettier-ignore -->
-Resource paths allowed:
+  @example link paths allowed:
   - 18f-accessibility-guide
   - /18f-accessibility-guide
   - /resources/18f-accessibility-guide
+  - https://wwww.ai.gov
+  - http://www.usps.com/
+
+  @example shortcode with internal link:
+  {{< featured-resource link = "/guides/hcd/discovery-concepts" >}}
```

Comment:
Can we remove these unnecessary spaces from examples?

```suggestion
  {{< featured-resource link="/guides/hcd/discovery-concepts" >}}
```